File: 151002/Volontirov/restApi/publisher/src/main/java/org/example/dc/model/Editor.java
package org.example.dc.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.validator.constraints.Length;

import javax.validation.constraints.Max;
import javax.validation.constraints.Min;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Editor {
    private int editor_id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 151002/Kuvshinov/lab2/Models/Editor.cs
namespace DC.Models
{
	public class Editor : BaseModel
    {
		public string Login { get; set; } = string.Empty;

        public string Password { get; set; } = string.Empty;

        public string Firstname { get; set; } = string.Empty;

        public string Lastname { get; set; } = string.Empty;
    }
}
